### PR TITLE
chore: improve README.md standardization

### PR DESCRIPTION
**Pull request by smcclellan**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: '@agraebe'
```
Comment (rafaelcr) [inline .github/ISSUE_TEMPLATE/feature_request.md:6]: Could you assign me instead?

File: README.md [hunk 2]
```diff
@@ -74,4 +77,35 @@ specified by the `RUN_MODE` environment variable:
...
 the service has time to finish any pending background work and all dependencies are gracefully
-disconnected.
\ No newline at end of file
+disconnected.
+
+## Bugs and feature requests
+
+If you encounter a bug or have a feature request, we encourage you to follow the steps below:
+
+ 1. **Search for existing issues:** Before submitting a new issue, please search [existing and closed issues](../../issues) to check if a similar problem or feature request has already been reported.
```
Comment (rafaelcr) [inline README.md:86]: Can you make sure these lines do not exceed 100 chars per line? There's a very handy VScode plugin I use called Rewrap that does this with 1 keystroke

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Bug'
+assignees: '@timstackblock'
```
Comment (rafaelcr) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Is Tim the right person to assign here? Just curious to see what your thought process is. I assume you'd like the Testing team to validate first?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (smcclellan) [reply .github/ISSUE_TEMPLATE/feature_request.md:6]: Better actually would be to keep this blank and create a CODEOWNERS in this repo to auto-assign someone.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (smcclellan) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: I'll remove this as well so it gets triaged as normal.

File: README.md (hunk 2 above)
Comment (smcclellan) [reply README.md]: Done.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (smcclellan) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Removed this line.

**Review by rafaelcr — APPROVED** [review]
Thanks @smcclellan !

**Pull request merged**